content/docs/authorization/deployment/helm/_index.md

```diff
@@ -106,6 +106,14 @@ To install CSM Authorization with the service Ingresses using a self-signed cert
 helm -n authorization install authorization -f myvalues.yaml charts/csm-authorization
 ```
 
+## Install Karavictl
+
+The Karavictl CLI can be obtained directly from the [GitHub repository's releases](https://github.com/dell/karavi-authorization/releases) section.
+
+In order to run `karavictl` commands, the binary needs to exist in your PATH, for example /usr/local/bin.
+
+Karavictl commands and intended use can be found [here](../../cli/) 
```

Comment:
Missing a period at the end of the sentence.

Reply:
fixed